twake/backend/node/src/core/platform/services/tracker/provider.ts

```diff
@@ -0,0 +1,30 @@
+import Analytics from "analytics-node";
+import { TwakeServiceProvider } from "../../framework";
+
+export type IdentityType =
+  | { userId: string | number }
```

Comment:
Do you need the first `|` ?

Reply:
This one is probably code formater, if yes you can let it like this @stephanevieira75

Reply:
Yes, it is code formater !